core/controllers/contributor_dashboard.py

```diff
@@ -361,54 +360,31 @@ def _get_reviewable_exploration_opportunity_summaries(
         Returns:
             list(ExplorationOpportunitySummary). A list of the matching
             exploration opportunity summaries.
-
-        Raises:
-            Exception. No exploration_id found for the node_id.
         """
-        # 1. Fetch the eligible topics.
-        # 2. Fetch the stories for the topics.
-        # 3. Get the reviewable translation suggestion target IDs for the user.
-        # 4. Get story exploration nodes in order, filtering for explorations
-        # that have in review translation suggestions.
-        if topic_name is None:
-            topics = topic_fetchers.get_all_topics()
-        else:
+        # 1. Fetch all the exploration IDs of chapters in stories under the
+        #    topic(s).
+        # 2. Get the reviewable translation suggestion target IDs for the user.
+        # 3. Get story exploration nodes in order, filtering for explorations
+        #    that have in review translation suggestions.
```

Comment:
This sentence doesn't scan and might need to be reworded.

Reply:
Done.